code/__HELPERS/math.dm

```diff
@@ -125,4 +125,5 @@
 	return TRUE
 
 
-
+/// rand() but for floats, returns a random floating point number between L and H
+#define randfloat(L, H) ((L) + rand() * ((H) - (L)))
```

Comment:
```suggestion
/// rand() but for floats, returns a random floating point number between low and high
#define randfloat(low, high) ((low) + rand() * ((high) - (low)))
```

Reply:
sure, just following the byond documentation ;)

https://www.byond.com/docs/ref/#/proc/rand